src/libs/antares/study/parts/hydro/container.cpp

```diff
@@ -799,10 +799,16 @@ double getWaterValue(const double& level /* format : in % of reservoir capacity
                      const Matrix<double>& waterValues,
                      const uint day)
 {
-    assert((level >= 0. && level <= 100.) && "getWaterValue function : invalid level");
+    assert((level >= 0. && level <= 100. + 1e-6) && "getWaterValue function : invalid level");
```

Comment:
assert are not triggered in release mode.
Maybe we should convert this to a proper error